mlflow/utils/virtualenv.py

```diff
@@ -220,10 +245,20 @@ def _get_or_create_virtualenv(local_model_path, env_id=None):
     local_model_path = Path(local_model_path)
     python_env = _get_python_env(local_model_path)
 
+    extra_env = _get_virtualenv_extra_env_vars(env_root_dir)
+    if env_root_dir is not None:
+        virtual_envs_root_path = Path(env_root_dir) / "virtualenv_envs"
+        pyenv_root_dir = os.path.join(env_root_dir, "pyenv")
```

Comment:
This line assumes `env_root_dir` contains `pyenv`. Do we copy `pyenv` in `env_root_dir` somewhere?

Reply:
@WeichenXu123 Does `_install_python` create this directory? Doesn't seem like we're creating it here.

Reply:
To clarify, the "pyenv_root_dir" is used to holds the python binaries downloaded by pyenv. We can still use "pyenv" installed in other directory .

> Does _install_python create this directory? Doesn't seem like we're creating it here.
I added `pyenv_root_dir.mkdir(parents=True, exist_ok=True)` to create it. (Don't create it is also fine, pyenv will automatically create it if it does not exist. But create it in advance is safer)

Reply:
Makes sense now. Can you add a comment above https://github.com/mlflow/mlflow/pull/5676/files#diff-addd2cf35dec6d4143b239c1a5b26d2a24d2a93540c3d4555ab289c1667b4017R117?

Reply:
add it in `def _install_python(version, pyenv_root=None, capture_output=False)` arugment doc